src/Kafka/CreateKafkaInstance/CreateKafkaInstance.tsx

```diff
@@ -128,6 +136,8 @@ export const CreateKafkaInstance: FunctionComponent<
   const disableAZTooltip =
     azOptions === undefined || (azOptions?.multi === true && azOptions.single);
 
+  if (!isRegionsAvailable) instanceAvailability = "trial-unavailable";
```

Comment:
I'm ok with the changes but please move them into the hook. The idea is to have all the business logic in the hook and the UI as a simple consumer of that. This way you don't have to call the hook twice just to make a const a let

Reply:
done